### DISCUSSION TITLE
Using java springboot in AKS , enabled istio mesh, how to do service to service invocation. And enable mtls

### DISCUSSION THREAD

**Discussion by krsujeetviet** [istio/istio#51244] [Q&A]
Trying to figure out how to do service to service invocation using istio in AKS, currently what is working for me is 
String target = "service-name:8086"; 
Which could be done with k8s dns.
Also please help with mtls enabling .

**Comment by rootsongjc** [comment]
[Here](https://github.com/eitansuez/spring-petclinic-istio) is a Spring on Istio tutorial for your reference.
